setup.py

```diff
@@ -65,12 +39,6 @@ def pip(filename):
     include_package_data=True,
     python_requires='>=3.7',
     install_requires=install_requires,
-    setup_requires=['setuptools>=38.6.0'],
-    tests_require=tests_require,
```

Comment:
Why don't we need those requirements anymore?

Reply:
`tests_require` is useful for `python setup.py test` AFAIK, and we don't use or document that. I could leave this one though, it's pretty standard.

`setup_requires` is not necessary anymore AFAIK, and even discouraged in some cases. I think it should be handled by the "host" system (CI or local computer), like for `wheel`.